src/checkout-buttons/strategies/masterpass/masterpass-button-strategy.spec.ts

```diff
@@ -75,10 +76,13 @@ describe('MasterpassButtonStrategy', () => {
             new FormFieldsActionCreator(new FormFieldsRequestSender(requestSender))
         );
 
+        locale = 'en-US';
```

Comment:
Instead of assigning to a temporary variable, you could directly pass it to the function